runtime/nls/cfre/cfrerr.nls

```diff
@@ -1668,3 +1668,20 @@ J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.explanation=Please consult the Java Virtual Ma
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.user_response=Contact the provider of the class file for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_MAJOR_VERSION2=%2$.*1$s has been compiled by a more recent version of the Java Runtime (class file version %3$u.%4$u), this version of the Java Runtime only recognizes class file versions up to %5$u.0; offset=%6$u
```

Comment:
This message can't be copied from Hotspot, pls make up a new one.

Reply:
Keeping the format of the original message:
bad major version; class=<class>, offset=<offset>. major.minor=<major>.<minor>, maximum major value=<max>.0

Reply:
Corrected the previous comment.